### DISCUSSION TITLE
How to add image and video with the help of Serverpod?

### DISCUSSION THREAD

**Discussion by someshwar16** [serverpod/serverpod#1579] [Q&A] [answered]
Hi, I am Someshwar I am currently making a Social Media App using Serverpod. Currently, I am making an Add Post section where I want to upload images as well as videos. I need help like how to upload a video and retrieving it from the database. Do we require an S3 bucket?

**Comment by Frank1016** [comment, marked as answer]
Yes, S3 is the way to go.

**Comment by someshwar16** [comment]
Thank You @Frank1016

**Comment by someshwar16** [comment]
@Frank1016 Can you help in configuring the S3 bucket? Because I am configuring it but my files are not uploading in S3.

**Reply by Frank1016** [reply]
Try using the amplify package from AWS:
https://docs.amplify.aws/flutter/start/getting-started/introduction/
Start from here

**Reply by someshwar16** [reply]
Can we integrate AWS Amplify and Serverpod together?

**Reply by Frank1016** [reply]
Yes, no problem at all
